entities/humans_swat.xml

```diff
@@ -0,0 +1,106 @@
+<Entities>
+	<Entity name="Pointman" type="Human" editorAutoHeight="false">
+		<RenderObject3D model="data/models/humans/ranger_assaulter.khm" diffuseTex="data/models/humans/SWAT_PlaceHolder.dds"/>
+		<RenderObject3D model="data/models/humans/attachments/ranger_assaulter_vest.khm" diffuseTex="data/models/humans/attachments/ranger_assaulter_vest.dds"/>
+		<RenderObject3D model="data/models/humans/attachments/ranger_assaulter_helmet.khm" diffuseTex="data/models/humans/attachments/ranger_assaulter_helmet.dds" attachSlot="Bip001 Head">
+			<Specular shininess="8" intensity="1.3"/>
+		</RenderObject3D>
+		<Breakable template="GenericTrooperGibs" breakOnDamage="explosive" deleteOnDeath="false"/>		
+		<PhysicalParams health="100"/>
+		<Human type="GoodGuy"
+			unit="SWAT"
+			class="Pointman">
+
+			<Id name="Pointman" portrait="data/textures/portraits/ranger2.dds" gender="0" voicePack="male_very_american"/>
+			<FOV degrees="90" distanceMeters="999" eyeRadiusMeters="0.6"/>
+			<Brain suppressionRecovery="30.0"/>
+			<Mobility>
+				<MoveSpeed min="1.1" defaultMetersPerSec="2.28" max="10"/>
+				<TurnSpeed min="6" defaultMetersPerSec="13" max="20"/>
+			</Mobility>
+
+			<Equipment>
+				<Item name="M9 Pistol"/>
+				<Item name="IronSights_Pistol"/>
+				<Item name="Flashbang"/>
+				<Item name="Slap Charge"/>
+				<Item name="Spy Camera"/>
+				<Item name="Crowbar"/>
+				<Item name="HelmetACH"/>
+				<Item name="Rangers SFLCS Level III"/>
+			</Equipment>
+		</Human>
+	</Entity>
+
+    <Entity name="SWAT Assaulter" type="Human" editorAutoHeight="false">
+		<RenderObject3D model="data/models/humans/ranger_assaulter.khm" diffuseTex="data/models/humans/SWAT_PlaceHolder.dds"/>
+		<RenderObject3D model="data/models/humans/attachments/ranger_assaulter_vest.khm" diffuseTex="data/models/humans/attachments/ranger_assaulter_vest.dds"/>
+		<RenderObject3D model="data/models/humans/attachments/ranger_assaulter_helmet.khm" diffuseTex="data/models/humans/attachments/ranger_assaulter_helmet.dds" attachSlot="Bip001 Head">
+			<Specular shininess="8" intensity="1.3"/>
+		</RenderObject3D>
+		<Breakable template="GenericTrooperGibs" breakOnDamage="explosive" deleteOnDeath="false"/>		
+		<PhysicalParams health="100"/>
+		<Human type="GoodGuy"
+			unit="SWAT"
+			class="AssaulterSW">
```

Comment:
For Classes I would keep them Long to avoid Ambiguity "SWAT Assaulter" or "Assaulter SWAT", just make it consistent with other Entries